pkg/subctl/cmd/gather/resource.go

```diff
@@ -66,3 +67,11 @@ func ResourcesToYAMLFile(info *Info, ofType schema.GroupVersionResource, namespa
 		info.Status.QueueFailureMessage(fmt.Sprintf("Failed to gather %s: %s", ofType.Resource, err))
 	}
 }
+
+func configMaps(info *Info, namespace string, listOptions metav1.ListOptions) {
```

Comment:
Since this is not accessed with the package specifier `gather.`,  we should prefix the function name for clarity.

```suggestion
func gatherConfigMaps(info *Info, namespace string, listOptions metav1.ListOptions) {
```